conf/security_events.conf.defaults

```diff
@@ -71,7 +72,20 @@ enabled=N
 [1100004]
 desc=Ancient OS isolation example
 template=banned_os
-trigger=DEVICE::28,DEVICE::29,DEVICE::30
+trigger=DEVICE::28,DEVICE::29,DEVICE::30,DEVICE::31,DEVICE::32
```

Comment:
Remove DEVICE::31 and DEVICE::32, they don't exist anymore, the configuration in the .example was faulty